core/src/edu/cornell/gdiac/main/CollisionController.java

```diff
@@ -1,32 +1,132 @@
 package edu.cornell.gdiac.main;
 
+import com.badlogic.gdx.physics.box2d.Body;
 import com.badlogic.gdx.physics.box2d.Contact;
+import com.badlogic.gdx.physics.box2d.ContactImpulse;
+import com.badlogic.gdx.physics.box2d.ContactListener;
+import com.badlogic.gdx.physics.box2d.Fixture;
+import com.badlogic.gdx.physics.box2d.Manifold;
+import com.badlogic.gdx.utils.ObjectSet;
+import edu.cornell.gdiac.physics.obstacle.BoxObstacle;
+import edu.cornell.gdiac.physics.obstacle.Obstacle;
 
-public class CollisionController {
+public class CollisionController implements ContactListener {
```

Comment:
Awesome job finding this `ContactListener` interface! It's great that you were able to completely remove collision logic from gameplay controller.